test/e2e/auth/pod_security_policy.go

```diff
@@ -270,6 +271,9 @@ func privilegedPSP(name string) *policy.PodSecurityPolicy {
 			RunAsUser: policy.RunAsUserStrategyOptions{
 				Rule: policy.RunAsUserStrategyRunAsAny,
 			},
+			RunAsGroup: &policy.RunAsGroupStrategyOptions{
+				Rule: policy.RunAsGroupStrategyMustRunAs,
```

Comment:
MustRunAs  should specify the id range the pod should run as and verify that pod in the range works without error and a pod outside that range should fail

Reply:
Updated the comments.

Reply:
thanks , looks good, could you add one more test for negative case, when mustrunas is 999 and pod spec is 1000 , then create pod fails ?

Reply:
agree we need a negative test that verifies a pod creation is rejected because of the runAsGroup field (ideally one testing runAsGroup in the spec.securityContext, and one testing runAsGroup in the spec.containers[0].securityContext)

Reply:
Sure, I will update the comments soon.